bottlecap/src/event_bus/bus.rs

```diff
@@ -1,20 +1,21 @@
-use std::sync::mpsc;
-use std::sync::mpsc::Sender;
+use std::sync::mpsc::{self, SyncSender};
 
 use crate::events;
 
+use super::constants::MAX_EVENTS;
```

Comment:
why `super` and not `crate::constants::MAX_EVENTS`?

Reply:
This is just how my editor imported it, but I think we should use crate here in case we want to move the constants file somewhere else in the future. Will change.